src/utils/warnTooManyClasses.js

```diff
@@ -0,0 +1,27 @@
+import styleSheet from '../models/StyleSheet'
+
+const components = WeakMap ? new WeakMap() : undefined
```

Comment:
Do we need this to be a WeakMap? Does Babel transpile that to an object or something or would this hurt browser support?